mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -21,230 +29,280 @@ instances:
-      ## Set extra_status_metrics to false to disable extra status metrics.
-      ## See the Datadog-MySQL metrics section to learn more:
-      ## https://docs.datadoghq.com/integrations/mysql/#metrics
-      #
-      # extra_status_metrics: true
-
-      ## @param extra_innodb_metrics - boolean - optional - default: true
-      ## Set extra_innodb_metrics to false to disable extra innodb metrics.
-      ## See the Datadog-MySQL metrics section to learn more:
-      ## https://docs.datadoghq.com/integrations/mysql/#metrics
-      #
-      # extra_innodb_metrics: true
-
-      ## @param schema_size_metrics - boolean - optional - default: false
-      ## Set schema_size_metrics parameter to collect schema size metrics. Note that it runs a heavy query
-      ## against your DB to compute the relevant metrics for all your existing schemas.
-      ## Due to the nature of these calls, if you have a high number of tables and schemas,
-      ## you may be subject to some negative impact in the performance of your DB.
-      ## See the Datadog-MySQL metrics section to learn more:
-      ## https://docs.datadoghq.com/integrations/mysql/#metrics
-      #
-      # schema_size_metrics: false
-
-      ## @param disable_innodb_metrics - boolean - optional - default: false
-      ## Set disable_innodb_metrics to true only if you are using older (unsupported) versions of
-      ## MySQL who do not run/have innodb engine support and may experiment issue otherwise.
-      ## Should this flag be enabled you will only receive a small subset of metrics.
-      ## See the Datadog-MySQL metrics section to learn more:
-      ## https://docs.datadoghq.com/integrations/mysql/#metrics
-      #
-      # disable_innodb_metrics: false
-
-      ## @param extra_performance_metrics - boolean - optional - default: true
-      ## extra_performance_metrics is reported if `performance_schema` is enabled
-      ## in the MySQL instance and if the version for that instance is >= 5.6.0
-      ##
-      ## extra_performance_metrics runs a heavy query against your DB to compute the relevant metrics
-      ## for all your existing schemas.
-      ## Due to the nature of these calls, if you have a high number of tables and/or schemas,
-      ## you may be subject to some negative impact in the performance of your DB.
-      ## Please bear that in mind when enabling them.
-      ## Metrics provided by the options:
-      ##   - mysql.info.schema.size (per schame)
-      ##   - mysql.performance.query_run_time.avg (per schema)
-      ##   - mysql.performance.digest_95th_percentile.avg_us
-      ##
-      ## Note that some of these require the user defined for this instance
-      ## to have PROCESS and SELECT privileges. Take a look at the
-      ## MySQL integration tile in the Datadog WebUI for further instructions.
-      #
-      # extra_performance_metrics: true
-
-## Log Section (Available for Agent >=6.0)
+    options:
```

Comment:
(nit) Shall we keep that commend by default ? I guess doesn't change much to have an empty dict either.

Reply:
Yes I agree, but `ddev` doesn't seem to support it for now. I agree an empty `options` dict by default shouldn't be a problem since we do

```python
options = instance.get('options', {}) or {}  # options could be None if empty in the YAML
```

I can look at how to add commented-out-by-default sections to `ddev`, but looks like it's not a blocker to get this PR merged.

Reply:
Yes, not a blocker for this PR.